src/css/profile/mobile/common/calendar.less

```diff
@@ -111,6 +105,10 @@ tr {
 		mask-position: center;
 		mask-size: 100%;
 		background-color: var(--calendar-arrow-color);
+
+		&.nonSelectionArrow {
```

Comment:
Classes for framework must be prefixed by "ui-".

Reply:
done.